recipes/triqs_dft_tools/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: triqs_dft_tools
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/dft_tools/releases/download/{{ version }}/dft_tools-{{ version }}.tar.gz
+  sha256: 9836b2e4ef1ba6fef2d9276e0a43cf13a68b1626f0e4fae312fba5f7c888a49a
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+  merge_build_host: True
+
+requirements:
+  build:
+    - cmake
+    - make
+    - clangxx >=8  # [osx]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - triqs 2.2
+  run:
+    - triqs 2.2
+
+test:
+  imports:
+    - triqs_dft_tools
+
+about:
+  home: https://triqs.github.io/dft_tools
+  license: GPLv3
+  license_file: LICENSE.txt
```

Comment:
Please add a license_family entry. CC @conda-forge/core This would have been caught by https://github.com/conda-forge/conda-smithy/pull/1256

Reply:
Done